apps/roam/src/utils/initializeObserversAndListeners.ts

```
@@ -0,0 +1,164 @@
+    className: "rm-graph-view-control-panel__main-options",
+    callback: (el) => {
+      const div = el as HTMLDivElement;
+      renderGraphOverviewExport(div);
+    },
+  });
+
+  if (isFlagEnabled("preview")) addPageRefObserver(previewPageRefHandler);
+  // if (isFlagEnabled("grammar.overlay")) {
+  //   addPageRefObserver((s) => overlayPageRefHandler(s, onloadArgs));
+  // }
+  if (!!getPageRefObserversSize()) enablePageRefObserver();
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)